### PR TITLE
Data Recording

### PR DESCRIPTION
**Pull request by jrvollmer**
Data recording backend, UI integration, and Dockerization

### CODE REVIEW

File: Backend/src/routes/api.js [hunk 1]
```diff
@@ -16,6 +28,145 @@ ROUTER.get("/api", (req, res) => {
...
+
+let sessionsList = [];
+
+const broadbandLines = new nReadlines(SESSIONS_LIST_PATH);
+let line;
+let lineNumber = 1;
+
+// Getting all the created sessions from sessionsList.bin
+while (line = broadbandLines.next()) {
+  // console.log(`Line ${lineNumber} has: ${line.toString('ascii')}`);
```
Comment (VIkranthMandadi) [inline Backend/src/routes/api.js]: Should this still be here or do we want to remove commented out code?

File: Backend/src/routes/api.js [hunk 2]
```diff
@@ -59,10 +210,20 @@ function openSocket() {
 
   // Data received listener
   client.on("data", (data) => {
-    // console.log(data);
-    console.time("update data");
-    unpackData(data);
-    console.timeEnd("update data");
+    if(data.length === bytesPerPacket) {
+      // console.log(data);
```
Comment (VIkranthMandadi) [inline Backend/src/routes/api.js]: Commented out code, not sure if this is necessary.

**Review by sarthyparty — APPROVED** [review]
LGTM

**Review by VIkranthMandadi — CHANGES_REQUESTED** [review]
The only comments I had were removing commented out code.

File: Backend/src/routes/api.js (hunk 2 above)
Comment (jrvollmer) [reply Backend/src/routes/api.js]: Addressed

File: Backend/src/routes/api.js (hunk 1 above)
Comment (jrvollmer) [reply Backend/src/routes/api.js]: Addressed

**Review by VIkranthMandadi — APPROVED** [review]
Looks good

**Pull request merged**